auth/src/main/java/com/firebase/ui/auth/ui/email/RecoverPasswordActivity.java

```diff
@@ -118,10 +118,21 @@ public void onClick(View view) {
     @Override
     public void onDonePressed() {
         if (mEmailFieldValidator.validate(mEmailEditText.getText())) {
-            mHandler.startReset(mEmailEditText.getText().toString());
+            if (getFlowParams().passwordResetSettings != null)
```

Comment:
Just a style nit: please use `{` and `}` for all blocks, even one-line blocks.  This is more consistent with the rest of the codebase and reduces bugs.

Reply:
Should I keep the functions `normalResetPassword` and `customResetPassword` since now there is only one function  `startReset` ?

Reply:
Good point, I think we can just have one function there too.